### DISCUSSION TITLE
A network worth looking into for DDS?

### DISCUSSION THREAD

**Discussion by Jonas-Soevik** [life-itself/community#1032] [General]
Trans European Halles connects people taking abandoned spaces and making them into creative, artistic and thriving places.
Sounds like they have the spirit to become deliberately developmental!

https://www.teh.net/?fbclid=IwAR05KE7dZ3E5mce5V4DIIjgTEnjNwbK9NAmHMgQdupy6PhIJucN_1q2BHhg

**Comment by rufuspollock** [comment]
@Jonas-Soevik thanks for the link! Very interesting.
